test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -91,13 +94,38 @@
     /**
      * Convert the entity from a {@link Response} into a map of maps.
      */
-    public Map<String, Object> entityAsMap(Response response) throws IOException {
+    public static Map<String, Object> entityAsMap(Response response) throws IOException {
```

Comment:
I made this static so I could make `hasXPack` static. There was no need for it to use the fancy registries provided by `createParser`.